scripts/ovpsim_log_to_trace_csv.py

```diff
@@ -55,6 +55,11 @@ def convert_mode(pri, line):
         "a0","a1","a2","a3","a4","a5","a6","a7",
         "s2","s3","s4","s5","s6","s7","s8","s9","s10","s11",
         "t3","t4","t5","t6"]
+FREGS = ["ft0","ft1","ft2","ft3","ft4","ft5","ft6","ft7","fs0",
+        "fs1",
```

Comment:
One more space here. Besides, make the line length more uniform.

Reply:
ok